src/lib.rs

```diff
@@ -69,16 +90,19 @@ pub fn run(config: &mut Config) -> Result<(), Box<dyn Error>> {
 pub struct Config {
     pub secret_word: String,
     pub correctly_guessed_letters: String,
+    pub lives: u32,
 }
```

Comment:
[nit]: Seeing as we are obviously writing this code for high-performance environments, I think we could slim this `Config` down slightly by using a `u8` for our `lives` counter. This should suffice until we start handing out more than 256 lives.